condDatasetSubmitter.py

```diff
@@ -77,10 +79,18 @@ def createOptionParser():
                      
   (options,args) = parser.parse_args()
 
-#  if not options.gt or not options.run or not options.conds:
-#      parser.error("options --run, --gt and --conds are mandatory")
-  if not options.newgt or not options.gt or not options.run:
-      parser.error("options --newgt, --run, and --gt  are mandatory")
+ #  print "** options args"
+ #  print options
+ #  print args
+ #  print "** options args"
+
+
+  if not options.newgt or not options.gt or not ( options.run or options.runLs):
+      parser.error("options --newgt, --run, [ either: --run  or: --runLs ] and --gt  are mandatory")
+
+  if (options.runLs):
+    options.runLs = ast.literal_eval(options.runLs)
```

Comment:
hi Antanas, thanks for the remark.
I've addressed it by replacing eval with ast.literal_eval wherever it made sense

I've tested locally, w/o a test submission of a request, will do in the coming days and report here